main.tf

```diff
@@ -5,7 +5,7 @@ locals {
   watsonx_orchestrate_guid          = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].guid : ibm_resource_instance.watsonx_orchestrate_instance[0].guid
   watsonx_orchestrate_name          = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].resource_name : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_name
   watsonx_orchestrate_plan_id       = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_plan_id
-  watsonx_orchestrate_dashboard_url = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].dashboard_url
+  watsonx_orchestrate_dashboard_url = var.existing_watsonx_orchestrate_instance_crn != null ? null : "https://us-south.watson-orchestrate.cloud.ibm.com/home/ws?plan_id=${urlencode(local.watsonx_orchestrate_plan_id)}"
```

Comment:
This is incorrect. It should not be `plan_id`. There can be various `wxO` instances and this will then become a generic URL for those instances with specific plan.
This has to be CRN. Were you able to test it manually?